app/templates/components/events/view/overview/event-apps.hbs

```diff
@@ -4,10 +4,14 @@
 <div class="content ui stackable grid">
   <div class="row">
     <div class="column eight wide center aligned">
-      <button class="ui button">{{t 'Generate Android App'}}</button>
+      <a href="{{settings.androidAppUrl}}" class="ui button">
+        {{t 'Generate Android App'}}
+      </a>
     </div>
     <div class="column eight wide center aligned">
-      <button class="ui button">{{t 'Generate Web App'}}</button>
+      <a href="{{settings.webAppUrl}}" class="ui button">
```

Comment:
No. This button should make an API call to the webapp/android app generator API and generate the app. Thats what this is for. Not a link to the generator.